site/content/entry/yuzu-progress-report-apr-2020/index.md

```diff
@@ -0,0 +1,211 @@
++ The Legend of Zelda: Link's Awakening - Miniature placing fixed
++ Toki Tori, Final Fantasy 7, Diablo 3, Project DIVA MegaMix - Distorted audio fixed
++ Team Sonic Racing - Physics fixed
++ Onimusha Warriors, Tales of Vesperia, The Messenger - Game logic fixed
++ Skyrim - Audio looping fixed, game progresses further
++ Oninaki - Elevator softlock fixed
++ All Unreal Engine 4 games - Motion jitter fixed
++ Super Smash Bros. Ultimate - Fighter select jitter / flickers fixed
++ Starlink - Softlock fixed, game progresses further
+
+&nbsp;
+
+{{< imgs
+    "./32.png| This sofa is no longer our enemy. (Pokémon Sword/Shield)"
+  >}}
+
+## Bug fixes and improvements
+[Morph](https://github.com/Morph1984) did some Kirby magic and implemented the first steps to get World of Light 
+working in `Super Smash Bros. Ultimate`. 
+Even thou it doesn’t work yet, the first step is always the most important one!
+
+[Kewlan](https://github.com/Kewlan) once again strikes with more UI quality of life improvements. 
+This time, more UI fixes to the placement of buttons mappings in the input settings, specifically this time for single joycons.
+
+{{< imgs
+    "./03.png| From this."
+  >}}
+  
+{{< imgs
+    "./05.png| To this."
+  >}}
+
+Not stopping there, Kewlan added a slider for the analog stick modifier, allowing keyboard users to half press the emulated 
+sticks. Plus now the sliders show percentages, making them easier to read.
+
+{{< imgs
+    "./07.png| Precision."
+  >}}
+
+And to top it off, he also added a way to restore the default value for a hotkey. Part of emulation is having a tidy interface 
+to set everything up.
+
+{{< imgs
+    "./08.png| More customization."
+  >}}
+
+[Tobi](https://github.com/FearlessTobi) implemented some fixes to the internal working of the input settings and the game list, 
+besides that, he also changed some of our error and warning windows to something more user friendly, thanks Tobi!
+
+{{< single-title-imgs
+    "The old errors."
+    "./19.png"
+    "./21.png"
+  >}}
+  
+{{< single-title-imgs
+    "The new ones."
+    "./20.png"
+    "./22.png"
+  >}}
+
+Waluigi is not part of Smash, yet, but that doesn’t mean his own tennis game has to look bad. 
+[Rodrigo](https://github.com/ReinUsesLisp) decided to help the purple fellow and fixed the shadows in `Mario Tennis Aces`. 
+The border color of textures in this game are in sRGB format, and we had some small precision errors. 
+Now you can enjoy those intensive matches!
+
+{{< single-title-imgs
+    "WAA! (Mario Tennis Aces)"
+    "./09.png"
+    "./10.png"
+  >}}
+
+[Rodrigo](https://github.com/ReinUsesLisp) added indexed QUADS to Vulkan, this brings mayor graphical fixes to `Xenoblade Chronicles 2` and `Fast RMX`! Some barrier fixes are still needed in `Xenoblade Chronicles 2` to get proper rendering, work on these rendering fixes is on-going.
+
+{{< single-title-imgs
+    "From abstract to RPG. (Xenoblade Chronicles 2)"
```

Comment:
Since the image for XC2 looks 99% black, might be appropriate to say `From black void to RPG` instead of `abstract`